packages/react-reconciler/src/__tests__/ReactSuspenseList-test.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
```

Comment:
This mattered because without the `node` environment it didn't have `performance.measure`. However, I now feature test it too.